CocoaLumberjack.podspec

```diff
@@ -32,6 +32,6 @@ Pod::Spec.new do |s|
 
   s.subspec 'Swift' do |ss|
     ss.dependency 'CocoaLumberjack/Core'
-    ss.source_files = 'Sources/CocoaLumberjackSwift/**/*.swift', 'Sources/CocoaLumberjackSwiftSupport/include/**/*.{h}'
+    ss.source_files = 'Sources/CocoaLumberjackSwift/**/*.swift',  'Sources/CocoaLumberjackSwiftLogBackend/**/*.swift', 'Sources/CocoaLumberjackSwiftSupport/include/**/*.{h}'
```

Comment:
This won't work. Users of the normal Swift subspec will get build failures due to the missing swift-log dependency.
Also, they are completely separate modules.

It would have to be a separate subspec which also declares the swift-log dependency correctly.

Reply:
Ah, I understand. I'll update the PR.